### ISSUE TITLE
The results of testing on the pre-trained model

### ISSUE THREAD

**Issue by buptmjj** [UT-Austin-RPL/Ditto#1]
Hello, we have tested using the pre-trained model provided in this paper, but the results seem to be incorrect. Where may there be a problem? We haven't changed the program code during this period. Thank you for your reply.
![Ditto_testresult](https://user-images.githubusercontent.com/87814885/157012230-f2f03a39-4099-4f90-9479-191898bc9385.png)

**Comment by Steve-Tod** [comment]
Hi, are you using Ditto_s2m.ckpt? Can you also provide the command you used to run this test?

**Comment by Steve-Tod** [comment]
Hi, thank you for pointing it out! I found I uploaded the wrong model. Now I updated the link for Ditto_s2m, and it should give performance close to the ones reported in the paper.

**Comment by buptmjj** [comment]
Hi, thank you very much for your reply. We have obtained the correct results through your latest pre-trained model. But when we test with our own training model, there seem to be some errors, such as cd_mobile. During the training, we changed the batch_size from 8 to 4 because of GPU. We are trying to find out the reason, but we also hope to get your help. Thank you very much.
![train-test-bestresult屏幕截图](https://user-images.githubusercontent.com/87814885/157196267-9903ecb9-ff2a-4242-a012-2e7dfbde9c8d.png)

**Comment by Steve-Tod** [comment]
Hmm, did you half the learning rate when changing the batch size?

**Comment by buptmjj** [comment]
Thank you very much for your reminder. I seem to know where the problem is!
